tools/rna_tools/rnalien/rnalien.xml

```diff
@@ -0,0 +1,119 @@
+<tool id="RNAlien" name="RNAlien" version="1.3.5">
+  <description>
+    Unsupervised covariance model construction from a single input sequence
+  </description>
+  <requirements>
+    <requirement type="package" version="1.3.5">rnalien</requirement>
+  </requirements>
+  <stdio>
+    <exit_code range="1:" />
+  </stdio>
+  <command><![CDATA[
+    RNAlien 
+      -i '$input_sequence'
+      #if $taxonomy_id
+                -t $taxonomy_id
+      #end if 
+      -o `pwd`
+      -d outdir
+      -c "\${GALAXY_SLOTS:-4}"
+      -z $z
+      -e $e
+      -b $b
+      $a
+      #if $tax_restriction
+           -r $tax_restriction
+      #end if         
+      #if $setup_check
```

Comment:
Is also not defined in the arguments. And I would not include it at all as an option.